### PR TITLE
Component: Post Body PR

### PR DESCRIPTION
**Pull request by torykling**
#73

The rough styling is there but it's hard to tell how close it is with the fonts being different. Tomorrow I'll figure out how to get the fonts imported correctly. 

Question about css class names - If I name a class name for this component .title or .date could the styles I apply affect anyone else's component with the same class name? I made all these class names post-body- something but I'll change it if they won't conflict. 

I wasn't sure how the data would look here for the props, so I guessed.

### CODE REVIEW

File: client/src/components/PostBody/PostBody.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
+.PostBody {
+  background: #fafafa;
+  width: 50rem;
+  font-family: "Open Sans", sans-serif;
+  color: #252525;
```
Comment (ZakkMan) [inline client/src/components/PostBody/PostBody.css]: Can you please use the CSS variables for color and font properties?

File: client/src/components/PostBody/PostBody.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+}
+
+.post-body-top-right {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+}
+
+.post-body-title {
+  font-family: "Reem Kufi", sans-serif;
```
Comment (ZakkMan) [inline client/src/components/PostBody/PostBody.css]: Can you use the CSS variable for the font name?

File: client/src/components/PostBody/PostBody.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import "./PostBody.css";
+
+// Function based React Component
+const PostBody = props => {
+  // Default Class to apply to Component
+  let classList = `PostBody`;
```
Comment (ZakkMan) [inline client/src/components/PostBody/PostBody.js]: If you're not updating or changing the class name, then you can just reference the class as a string in the JSX and remove this extra variable

File: client/src/components/PostBody/PostBody.css (hunk 1 above)
Comment (torykling) [reply client/src/components/PostBody/PostBody.css]: Sounds good. I asked Bill to add Open Sans to the app.css. Once his PR gets merged, I can add that variable here.

File: client/src/components/PostBody/PostBody.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import "./PostBody.css";
+import "../../App.css";
```
Comment (ZakkMan) [inline client/src/components/PostBody/PostBody.js]: You shouldn't need to import the `App.css` file anymore - it's been added to the storybook global config

File: client/src/components/PostBody/PostBody.css [hunk 5]
```diff
@@ -0,0 +1,49 @@
+.PostBody {
+  background: var(--white);
+  width: 50rem;
+  font-family: var(--paragraph-font);
+  color: var(--black);
+}
```
Comment (ZakkMan) [inline client/src/components/PostBody/PostBody.css]: I think you need some padding here

File: client/src/components/PostBody/PostBody.js (hunk 4 above)
Comment (torykling) [reply client/src/components/PostBody/PostBody.js]: Removed this. Bill's going to import the bold version of open sans.

**Pull request merged**